builtin/providers/aws/resource_aws_api_gateway_api_key.go

```diff
@@ -71,6 +76,7 @@ func resourceAwsApiGatewayApiKeyCreate(d *schema.ResourceData, meta interface{})
 		Description: aws.String(d.Get("description").(string)),
 		Enabled:     aws.Bool(d.Get("enabled").(bool)),
 		StageKeys:   expandApiGatewayStageKeys(d),
+		Value:       aws.String(d.Get("value").(string)),
```

Comment:
IF this value is Computed: true and then nothing else, a user cannot set it

therefore this will fail - does the `value` need to be optional as well?